README.md

```diff
@@ -42,6 +42,71 @@ and configure these applications.
+| **Red Hat Connectivitiy Link Operator** | Manage the lifecycle of the Kuadrant system | Optional - Install from OperatorHub |
+| **LeaderWorkerSet (LWS) Operator** | Leader-worker set controller for distributed training | Advanced distributed training workflows | Optional - Install from OperatorHub for Kueue integration |
+| **OpenTelemetry Operator** | Distributed tracing and telemetry | Monitoring and observability | Optional - for advanced monitoring |
+| **Tempo Operator** | Distributed tracing backend | Tracing infrastructure | Optional - for trace storage |
+| **Cluster Observability Operator** | Enhanced cluster monitoring | Production monitoring | Optional - for comprehensive observability |
+| **Perses Operator** | Dashboard and visualization for metrics | Monitoring dashboards and visualizations | Optional - Install from OperatorHub for enhanced metric visualization |
+
+
+#### ODH Component Operators (Managed by ODH)
+
+The following operators are **automatically deployed and managed** by the ODH operator based on your DataScienceCluster configuration:
+
+| Component | Operator/Repository | Purpose | Management State |
+|-----------|-------------------|---------|------------------|
+| **KServe** | [opendatahub-io/kserve](https://github.com/opendatahub-io/kserve) | Model serving platform | Optional |
+| **Ray** | [opendatahub-io/kuberay](https://github.com/opendatahub-io/kuberay) | Distributed computing framework | Optional |
+| **Training Operator** | [opendatahub-io/training-operator](https://github.com/opendatahub-io/training-operator) | ML training job management | Optional |
+| **Feast Operator** | [opendatahub-io/feast](https://github.com/opendatahub-io/feast) | Feature store for ML | Optional |
+| **Model Registry** | [opendatahub-io/model-registry](https://github.com/opendatahub-io/model-registry) | Model versioning and registry | Optional |
+| **TrustyAI** | [opendatahub-io/trustyai-service-operator](https://github.com/opendatahub-io/trustyai-service-operator) | AI explainability and governance | Optional |
+| **Dashboard** | [opendatahub-io/odh-dashboard](https://github.com/opendatahub-io/odh-dashboard) | Web UI for ODH management | Optional |
+| **Workbenches** | [opendatahub-io/notebooks](https://github.com/opendatahub-io/notebooks) | Jupyter notebook environments | Optional |
+| **Data Science Pipelines** | [opendatahub-io/data-science-pipelines-operator](https://github.com/opendatahub-io/data-science-pipelines-operator) | ML pipeline orchestration | Optional |
+
+**Note**: These component operators do **not** need to be installed separately. They are deployed and managed by the ODH operator based on your DataScienceCluster configuration.
+
+#### Optional Components
+
+- **GPU Support**: For GPU workloads and metrics:
+  - NVIDIA GPU Operator (for NVIDIA GPUs)
+  - NVIDIA DCGM Exporter (for GPU metrics collection)
```

Comment:
For other accelerators, are there equivalent operators, or is it only when it's NVIDIA that extra stuff is needed here?